components/capi/include/locale.h

```diff
@@ -0,0 +1,14 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+#pragma once
```

Comment:
As far as I know, `#pragma once` is not standardized and so is not something we should rely upon if we want to support as many different C compilers as possible, e.g. for embedded systems. It is also currently disallowed by the Mozilla style guide, although I think that is more for internal reasons of how we structure our headers.